lib/server.js

```diff
@@ -64,13 +74,27 @@ function Server(options) {
     console: console,
     server: this,
   });
-  var meshOptions = {};
+  this.minkelite = null;
```

Comment:
Doesn't even need to live on the object, the only references are inside this constructor.